db/migrate/20230207114821_drop_referenced_id_on_search_references.rb

```diff
@@ -0,0 +1,13 @@
+Sequel.migration do
+  up do
+    alter_table :search_references do
+      drop_column :referenced_id
+    end
+  end
+
+  down do
+    alter_table :search_references do
+      add_column :referenced_class, String, size: 10
```

Comment:
This should be `referenced_id` rather then `referenced_class`

Reply:
:facepalm: